### PR TITLE
add firebase-tools (firebase cli)

### PR DESCRIPTION
**Pull request by Vrtak-CZ**
It's required for using Firebase App Distribution.
https://firebase.google.com/products/app-distribution

### Pull Request Checklist

Check/accept these:

- [x] I've updated the version in the Dockerfile
- [x] I've updated the CHANGELOG.md
- [x] I've provided a link or explanation below which describes the changes in the tool itself

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -152,6 +152,16 @@ RUN /usr/bin/gcloud config set --installation component_manager/disable_update_c
...
+# ------------------------------------------------------
+# --- Install Firebase Tools (Firebase CLI)
+# https://github.com/firebase/firebase-tools
+#
+# It's required for using Firebase App Distribution.
+#  https://firebase.google.com/products/app-distribution
+#
+
+RUN npm install -g firebase-tools \
+ && firebase --version
```
[the remark this answers is not in the document]
Comment (Vrtak-CZ) [reply Dockerfile]: Fixed

**Comment by viktorbenei** [comment]
The usual concern with any preinstalled tool is whether they require the latest version of this tool to be available, cause if they do it’s usually not a good idea to preinstall the tool as that’ll only be updated once a week.

That said in this case we'll also have a Step for Firebase App Distribution which should be able to handle this case, when a newer version of the tool would be required and update the tool during the step's run.

**Comment by koral--** [comment]
According to Firebase CLI docs there is no strong requirement on having the latest version. It's only a good practice.
https://firebase.google.com/docs/cli#update-cli

**Comment by ghaabor** [comment]
@Vrtak-CZ please update the CHANGELOG.md file with the added tool and the version number accordingly in the Dockerfile

**Comment by viktorbenei** [comment]
I submitted our usual PR checklist to this repo https://github.com/bitrise-io/android/pull/260

**Comment by Vrtak-CZ** [comment]
> @Vrtak-CZ please update the CHANGELOG.md file with the added tool and the version number accordingly in the Dockerfile

Done

**Comment by viktorbenei** [comment]
Also thanks @koral-- for the infos ;)

**Pull request merged**